### PR TITLE
feat: implement eth_sendBundle and eth_bundlePrice

### PR DESCRIPTION
**Pull request by Pythonberg1997**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: core/txpool/legacypool/legacypool.go [hunk 1]
```diff
@@ -56,17 +56,9 @@ const (
...
-
-	// ErrTxPoolOverflow is returned if the transaction pool is full and can't accept
-	// another remote transaction.
-	ErrTxPoolOverflow = errors.New("txpool is full")
-
-	ErrInBlackList = errors.New("sender or to in black list")
-)
+// ErrTxPoolOverflow is returned if the transaction pool is full and can't accept
+// another remote transaction.
+var ErrTxPoolOverflow = errors.New("txpool is full")
```
Comment (irrun) [inline core/txpool/legacypool/legacypool.go:61]: why only leave this?

File: eth/api_backend.go [hunk 2]
```diff
@@ -294,6 +295,31 @@ func (b *EthAPIBackend) SendBundle(ctx context.Context, bundle *types.Bundle) er
...
+		return bundles[j].Price.Cmp(bundles[i].Price) < 0
+	})
+	gasFloor := big.NewInt(b.eth.config.Miner.MevGasPriceFloor)
+	idx := len(bundles) / 2
+	if bundles[idx].Price.Cmp(gasFloor) < 0 {
+		return gasFloor
+	}
+	return bundles[idx].Price
+}
+
```
Comment (irrun) [inline eth/api_backend.go:322]: too compact style, add some empty lines to separate paragragh?

File: core/txpool/legacypool/legacypool.go (hunk 1 above)
Comment (Pythonberg1997) [reply core/txpool/legacypool/legacypool.go:61]: Moved others to common errors

File: core/txpool/bundlepool/bundlepool.go [hunk 3]
```diff
@@ -50,40 +69,91 @@ type BundlePool struct {
...
+		return ErrBundleGasPriceLow
+	}
+	bundle.Price = price
+
+	pool.mu.Lock()
+	defer pool.mu.Unlock()
+	if _, ok := pool.bundles[hash]; ok {
+		return ErrBundleAlreadyExist
+	}
+	if len(pool.bundles) > int(pool.config.BundleSlot) {
```
[the remark this answers is not in the document]
Comment (Pythonberg1997) [reply core/txpool/bundlepool/bundlepool.go:127]: Removed

**Pull request merged**